doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20.rst

```diff
@@ -0,0 +1,439 @@
+  * Global Domain Frequency Scaling (GDFS), optimizing USB suspend power consumption.
+  * Drivers adapted for the following peripherals:
+
+    * UARTE120
+    * PWM120
+    * SPIM120
+    * EXMIF
+
+  * UICR validation support.
+  * MSPI EXMIF support.
+
+Added the following features as experimental:
+
+* Bluetooth®:
+
+  * :ref:`Parallel scanning and initiating connection <bt_scanning_while_connecting>`.
+    It reduces the overall waiting time when there are several Bluetooth devices to be discovered and connected at the same time.
+
+Improved the following features:
+
+* Better BICR handling and generation.
+* Enhanced power management features across various subsystems.
+
+Implemented the following bug fixes:
+
+* NCSDK-30802: Resolved an issue where the nRF54H20 device suddenly stopped transmitting ESB packets after nrfxlib 3.9.0.
+* NCSDK-30161: Fixed an assertion during boot time caused by a combination of :kconfig:option:`CONFIG_ASSERT`, :kconfig:option:`CONFIG_SOC_NRF54H20_GPD`, and external flash.
+* NCSDK-30117: Ensured that it is no longer possible to declare a MEM component pointing to a memory region not assigned to a particular core.
+* NCSDK-29682: Added support for the ``cose-alg-sha-512`` algorithm in the SUIT module.
+
+Limitations
+***********
+
+On the nRF54H20 SoC, the Device Firmware Update (DFU) procedure from external flash memory does not work with the new flash memory driver based on the MSPI EXMIF handling.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.9.0-nRF54H20**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.9.0-nrf54h20`_.
+
+Integration test results
+************************
+
+The integration test results for this tag can be found in the following external Artifactory:
+
+* `Twister test report for nRF Connect SDK v2.9.0-nRF54H20`_
+* `Hardware test report for nRF Connect SDK v2.9.0-nRF54H20`_
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.9.0-nrf54h20.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.9.0-nrf54h20`_ for the list of issues valid for the latest release.
+
+Migration notes
+***************
+
+See the `Migration guide for nRF Connect SDK v2.9.0-nrf54h20`_ for the changes required or recommended when migrating your nRF54H Series application from |NCS| v2.8.0 to |NCS| v2.9.0-nrf54h20.
```

Comment:
```suggestion
See the `Migration guide for nRF Connect SDK v2.9.0-nRF54H20`_ for the changes required or recommended when migrating your nRF54H Series application from |NCS| v2.8.0 to |NCS| v2.9.0-nRF54H20.
```

Reply:
This needs to be fixed in the "v2.9.0-nRF54H20-rc1 Release Notes" as well.

Reply:
rc-1 to be removed